packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -140,9 +140,10 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
         css={css`
           margin: ${euiTheme.size.xs};
           box-shadow: 0 0 4px 4px ${euiTheme.colors.darkShade};
+          height: 90%;
```

Comment:
is this `height` really necessary? I'm turning it on and off and can't see a difference...